### PR TITLE
[DOC]Remove outdated compatibility notices and rework cloud notice

### LINKED ISSUE #936: [Doc]Remove old compatibility notes
These notes don't make sense in 7.x (or 6.x for that matter). Documentation for older versions is still available if somebody needs it.

Screen Shot 2020-04-17 at 11 16 14 AM

Screen Shot 2020-04-17 at 11 16 22 AM

### PR DESCRIPTION
**Pull request by karenzone**
Fixes #936

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -21,6 +21,16 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+If you plan to use the Kibana web interface, use the Elasticsearch output plugin
+to get your log data into Elasticsearch. 
+
+This output only speaks the HTTP protocol. HTTP is the preferred protocol for interacting with Elasticsearch.
```
Comment (jsvd) [inline docs/index.asciidoc:27]: We can remove this paragraph as node protocol hasn’t been available for some time and we don’t provide transport protocol support anymore in Logstash.

File: docs/index.asciidoc [hunk 2]
```diff
@@ -21,6 +21,16 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+If you plan to use the Kibana web interface, use the Elasticsearch output plugin
```
Comment (jsvd) [inline docs/index.asciidoc:24]:
```suggestion
If you plan to use the Kibana web interface to analyze data transformed by Logstash, use the Elasticsearch output plugin
```

File: docs/index.asciidoc [hunk 3]
```diff
@@ -21,6 +21,16 @@ include::{include_path}/plugin_header.asciidoc[]
...
+to get your log data into Elasticsearch.
```
Comment (jsvd) [inline docs/index.asciidoc:25]:
```suggestion
to get your data into Elasticsearch. 
```

File: docs/index.asciidoc [hunk 4]
```diff
@@ -21,6 +21,16 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+If you plan to use the Kibana web interface to analyze data transformed by Logstash, use the Elasticsearch output plugin
+to get your data into Elasticsearch. 
+
+This output only speaks the HTTP protocol. HTTP is the preferred protocol for interacting with Elasticsearch.
+We strongly encourage the use of HTTP over the node protocol for a number of reasons. HTTP is only marginally slower,
+yet far easier to administer and work with. When using the HTTP protocol one may upgrade Elasticsearch versions without having
+to upgrade Logstash in lock-step.
```
Comment (jsvd) [inline docs/index.asciidoc:30]:
```suggestion
This output only speaks the HTTP protocol as it is the preferred protocol for interacting with Elasticsearch.
In previous versions it was possible to communicate with Elasticsearch through the transport protocol, which is now reserved for internal cluster communication between nodes https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-transport.html. This has been deprecated in Elasticsearch 7.0.0 and will be removed in 8.0.0 https://www.elastic.co/guide/en/elasticsearch/reference/current/java-clients.html
```

File: docs/index.asciidoc [hunk 5]
```diff
@@ -21,6 +21,20 @@ include::{include_path}/plugin_header.asciidoc[]
...
+If you plan to use the Kibana web interface to analyze data transformed by
+Logstash, use the Elasticsearch output plugin to get your data into
+Elasticsearch. 
+
+This output only speaks the HTTP protocol as it is the preferred protocol for
+interacting with Elasticsearch. In previous versions it was possible to
+communicate with Elasticsearch through the transport protocol, which is now
+reserved for internal cluster communication between nodes
+https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-transport.html.
+This has been deprecated in Elasticsearch 7.0.0 and will be removed in 8.0.0
```
Comment (karenzone) [inline docs/index.asciidoc:33]: @jsvd Thanks for adding this info. Can you clarify this?
"This" (in "This has been deprecated...") means  "transport protocol?"

Comment (jsvd) [reply docs/index.asciidoc:33]: Using the transport protocol to communicate with the cluster

Comment (karenzone) [reply docs/index.asciidoc]: Thanks for clarifying. Please see what you think.

**Comment by karenzone** [comment]
**PUBLISHED:**
Screen Shot 2020-04-30 at 11 10 36 AM

**Pull request merged**